aiida/manage/external/postgres.py

```diff
@@ -22,7 +22,7 @@
 
 __all__ = ('Postgres', 'PostgresConnectionMode', 'DEFAULT_DBINFO')
 
-_CREATE_USER_COMMAND = 'CREATE USER "{}" WITH PASSWORD \'{}\''
+_CREATE_USER_COMMAND = 'CREATE USER "{}" WITH PASSWORD \'{}\' {}'
```

Comment:
perhaps add a comment here what the last field is for (it's less obvious than for the other fields)

Reply:
done